packages/theme-chalk/src/tour.scss

```diff
@@ -0,0 +1,169 @@
+@use 'sass:map';
+
+@use 'mixins/mixins' as *;
+@use 'mixins/var' as *;
+@use 'common/var' as *;
+
+@include b(tour) {
+  @include set-component-css-var('tour', $tour);
+
+  @include e(content) { 
+    padding: 0;
+    border-radius: getCssVar('tour-border-radius');
+    border: 1px solid getCssVar('tour-border-color');
+    width: var(#{getCssVarName('tour-width')});
+    background: getCssVar('tour-bg-color');
+    box-shadow: getCssVar('box-shadow-light');
+    box-sizing: border-box;
+
+    $content-selector: &;
+
+    $sides: (
+      'top': 'bottom',
+      'bottom': 'top',
+      'left': 'right',
+      'right': 'left',
+    );
+
+    @include e(arrow) {
+      position: absolute;
+      background: getCssVar('tour-bg-color');
+      border: 1px solid getCssVar('tour-border-color');
+      width: 10px;
+      height: 10px;
+      pointer-events: none;
+      transform: rotate(45deg);
+      box-sizing: border-box;
+
+      @each $side,
+      $adjacency
+        in ('top': 'left', 'bottom': 'right', 'left': 'bottom', 'right': 'top')
+      {
+        #{$content-selector}[data-side^='#{$side}'] & {
+          border-#{$side}-color: transparent;
+          border-#{$adjacency}-color: transparent;
+        }
+      }
+
+      @each $side, $opposite in $sides {
+        #{$content-selector}[data-side^='#{$side}'] & {
+          #{$opposite}: -5px;
+        }
+      }
+    }
+
+
+    @include e(closebtn) {
+      position: absolute;
+      top: 6px;
+      right: 0;
+      padding: 0;
+      width: 44px;
+      height: 44px;
+      background: transparent;
+      border: none;
+      outline: none;
+      cursor: pointer;
+      font-size: var(
+        #{getCssVarName('message-close-size')},
+        map.get($message, 'close-size')
+      );
+
+      .#{$namespace}-tour__close {
+        color: getCssVar('tour-title-text-color');
+        font-size: inherit;
+      }
+
+      &:focus,
+      &:hover {
+        .#{$namespace}-tour__close {
+          color: getCssVar('color', 'primary');
+        }
+      }
+    }
+
+    @include e(cover) {
+      text-align: center;
+      padding: 46px 16px 0;
+      img, video {
+        width: 100%;
+      }
+    }
+
+    @include e(header) {
+      padding: getCssVar('tour', 'padding-primary');
+      padding-bottom: 8px;
+      margin-right: 16px;
+    }
+
+    @include e(title) {
+      line-height: getCssVar('tour-font-line-height');
+      font-size: getCssVar('tour-title-font-size');
+      color: getCssVar('tour-title-text-color');
+      font-weight: getCssVar('tour-title-font-weight');
+    }
```

Comment:
To ensure the same UI style
I suggest referring to popover
changing the padding-bottom to 10px
changing the font-size to 16px
changing the font-weight to 400